course/src/main/java/org/openedx/course/presentation/outline/CourseOutlineViewModel.kt

```diff
@@ -236,6 +245,24 @@ class CourseOutlineViewModel(
         return resumeBlock
     }
 
+    fun resetCourseDatesBanner(onResetDates: (Boolean) -> Unit) {
+        viewModelScope.launch {
+            try {
+                interactor.resetCourseDates(courseId = courseId)
+                onResetDates(true)
```

Comment:
We can move the snackbar to the Course tab scope and show the snackbar using `_uiMessage`. It was in course/home tab in market app as well.